tools/packaging/packaging.psm1

```diff
@@ -335,6 +335,24 @@ function Start-PSPackage {
                     New-MSIPackage @Arguments
                 }
             }
+            "msix" {
+                $TargetArchitecture = "x64"
```

Comment:
Do we need `arm64` here?

Reply:
As far as I know, the makeappx.exe tool is just creating a pkg so it doesn't matter what architecture the binaries are built for so there's no reason to even check `$TargetArchitecture`.  I'll update it so it will work essentially as a `zip` without needing to know the architecture other than naming the resulting msix.